sovryn-token-bridge/federator/src/main.js

```diff
@@ -54,12 +85,37 @@ const sideFederator = new Federator(
 );
 
 let pollingInterval = config.runEvery * 1000 * 60; // Minutes
+// let gasApiPollingInterval = config.gasApiRunEvery * 1000 ; // Seconds
+// let avgGasPollingInterval = config.avgGasRunEvery * 1000 ; // Seconds
+// let avgGasPeriodInterval = config.periodAvgGas * 1000 * 60 ; // Minutes
+// let avgGasCount;
+
 let scheduler = new Scheduler(pollingInterval, logger, { run: () => run() });
 
-scheduler.start().catch((err) => {
-    logger.error('Unhandled Error on start()', err);
-});
+startServices().catch(err => { console.error('Error starting services:', err) });
 
+async function startServices() {
+    try {
+        isEth = await clientId._getChainId();

```

Comment:
`clientId._getChainId` was renamed to `clientId.isEthereumChain`

Assigning to `isEth` without `const` or `let` creates a poorly-defined global variable -- it's better to do:

```
    let isEth;
    try {
        isEth = await clientId._getChainId();
 ```

Reply:
Done